src/components/Counter/Counter.css

```diff
@@ -0,0 +1,14 @@
+@value breakpoint, fontstack from 'styles/variables.css';
+@value black from theme(colors);
```

Comment:
`theme(colors)` is basically just short for `/styles/theme/default/colors.css`. but it means as you go on, you don't have to refine loads of includes when changing your theme (just change base path in webpack)